### DISCUSSION TITLE
Labeled and predicted over 700 instances and still not getting an accuracy score over 0.9

### DISCUSSION THREAD

**Discussion by tibbitts16** [talmolab/sleap#1394] [Help!]
Hello I hope this message meets you well. 

I have been using SLEAP to begin to score social interaction tests for mice. Before getting there I have been using SLEAP for animal pose estimation, I am currently on the "Prediction-assisted labeling' step. Up to this point I have just been running training under a multi-animal top down system and predicting on random frame( 20 total frames) Once the training is done I have been manually correcting the inferences that are below 0.9 scores and then repeating this process. Unfortunately, I have been unsuccessful in getting the scores for the inferences to collectively be above 0.9 and am unsure how to problem solve this issue.
Should I go ahead and run inferences on the entire video if the scores arent accurate enough, and how many predicted frames does it usually take for 90% accuracy and above to be achieved?

Thanks,

Tommy

**Comment by roomrys** [comment]
Hi @tibbitts16,

The scores assigned to each predicted instance are how confident SLEAP is that those predictions are correct, and while this might seem like a measure of accuracy, it is not an accuracy score. I wrote a little excerpt about all the scores SLEAP uses in #1268.

Aiming for all predictions to be above 0.9 might be a bit high in my opinion. Occasionally, SLEAP will output good pose estimations, but still assign a low confidence score. This score does not necessarily mean that the pose is bad or inaccurate, it just means that SLEAP is unsure if the pose is correct based on the examples it has been trained on. That being said, predictions do tend to be worse with lower confidence scores so that is a good rule of thumb for iterating through the prediction assisted labeling, but maybe not the best measure for determining when to stop labeling.

A better measure of how your model is performing would be to use the model metrics (Predict > Evaluate Model Metrics...). We also have a [notebook](https://sleap.ai/notebooks/Model_evaluation.html) to help with model evaluation. Unlike the prediction scores (which are based off of likelihood), the model metrics compare the distances of predicted points to points that the user has annotated and are a better measure of accuracy.

Thanks,
Liezl
